libdevcore/Exceptions.h

```diff
@@ -32,6 +32,9 @@
 #include "CommonData.h"
 #include "FixedHash.h"
 
+#pragma warning( push )
+#pragma warning(disable:811) //exception specification for implicitly declared virtual function
```

Comment:
The affect functions are listed below:
libdevcore/Exceptions.h(39): "dev::Exception::~Exception" 
libdevcore/Exceptions.h(51): "dev::RLPException::~RLPException" 
libdevcore/Exceptions.h(54): "dev::BadCast::~BadCast" 
libdevcore/Exceptions.h(55): "dev::BadRLP::~BadRLP"
libdevcore/Exceptions.h(56): "dev::OversizeRLP::~OversizeRLP" 
libdevcore/Exceptions.h(57): "dev::UndersizeRLP::~UndersizeRLP"
libdevcore/Exceptions.h(59): "dev::BadHexCharacter::~BadHexCharacter" 
libdevcore/Exceptions.h(60): "dev::NoNetworking::~NoNetworking"
libdevcore/Exceptions.h(61): "dev::NoUPnPDevice::~NoUPnPDevice" 
libdevcore/Exceptions.h(62): "dev::RootNotFound::~RootNotFound"
libdevcore/Exceptions.h(63): "dev::BadRoot::~BadRoot" 
libdevcore/Exceptions.h(64): "dev::FileError::~FileError"
libdevcore/Exceptions.h(65): "dev::Overflow::~Overflow" 
libdevcore/Exceptions.h(66): "dev::FailedInvariant::~FailedInvariant"
libdevcore/Exceptions.h(67): "dev::ValueTooLarge::~ValueTooLarge" 
libdevcore/Exceptions.h(69): "dev::InterfaceNotSupported::~InterfaceNotSupported"
libdevcore/Exceptions.h(70): "dev::ExternalFunctionFailure::~ExternalFunctionFailure"